### ISSUE TITLE
BibTeX Warning for Inbook Entries with Author and Editor Fields

### ISSUE THREAD

**Issue by qak** [retorquere/zotero-better-bibtex#1541]
I have a Book Section item in Zotero with fields Author, Book Author and Editor specified. BBT's BibTeX exporter outputs this item as an `inbook` entry with fields `author` (from Zotero's Author) and `editor` (from Zotero's Editor). When I pass the produced file to the bibtex program, the following warning is printed:

```
Warning--can't use both author and editor fields in [citation-key]
```

The produced `.bbl` file only contains the content of the entry's `author` field. According to BibTeX's [documentation](http://mirrors.ctan.org/biblio/bibtex/base/btxdoc.pdf), `inbook` entries should either contain an `author` field or an `editor` field, but not both.

> **inbook** A part of a book, which may be a chapter (or section or whatever) and/or a range of pages. Required fields: `author` or `editor`, `title`, `chapter` and/or `pages`, `publisher`, `year`. Optional fields: `volume` or `number`, `series`, `type`, `address`, `edition`, `month`, `note`.

Because the bibtex program prefers to use the `author` field instead of the `editor` field when an `inbook` entry has both, I think that BBT's BibTeX exporter should only output `inbook` entries with `author` (when both `author` and `editor` are available) to avoid the mentioned warning altogether. The report ID: `6479CHXA-euc`.

**Comment by retorquere** [comment]
Man, *thank you* for submitting an issue *with* a debug log. I can't believe that the template asks for it, label-gun asks for it, and even then people more often than not omit it. It makes my life so much easier.

**Comment by blip-bloop** [comment]
:robot: this is your friendly neighborhood build bot announcing [test build 5.2.28.6655](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-5.2.28.6655.xpi) ("fixes #1541")

Install in Zotero by downloading [test build 5.2.28.6655](https://github.com/retorquere/zotero-better-bibtex/releases/download/builds/zotero-better-bibtex-5.2.28.6655.xpi), opening the Zotero "Tools" menu, selecting "Add-ons", open the gear menu in the top right, and select "Install Add-on From File...".

**Comment by qak** [comment]
You're welcome 😉. The test build fixes the issue.
